### PR TITLE
Tweak: Optimize button widget markup [ED-14229]

### PR DESCRIPTION
**Pull request by rami-elementor**
Current button widget markup:

```html
<div class="elementor-button-wrapper">
	<a class="elementor-button elementor-button-link elementor-size-sm" href="#">
		<span class="elementor-button-content-wrapper">

			<span class="elementor-button-icon"> ... </span>
			<span class="elementor-button-text"> ... </span>

		</span>
	</a>
</div>
```

Optimized button widget markup:

```html
<a class="elementor-button elementor-button-link elementor-size-sm" href="#">
	<span class="elementor-button-content-wrapper">

		<span class="elementor-button-icon"> ... </span>
		<span class="elementor-button-text"> ... </span>

	</span>
</a>
```

### CODE REVIEW

File: tests/elements-regression/tests/templates/button.json [hunk 1]
```diff
@@ -1 +1 @@
...ove, padding all 10","button_text_color":"#030303","background_color":"#E1D626","border_border":"groove","text_padding":{"unit":"px","top":"10","right":"10","bottom":"10","left":"10","isLinked":true},"typography_typography":"custom","typography_font_family":"Roboto","typography_font_weight":"500"},"elements":[],"isInner":false,"widgetType":"button","elType":"widget"},{"id":"518be7b0","settings":{"text":"Gradient location 40, radial, position top left, boarder type double, width all 10, colour cusom #010102","background_background":"gradient","background_color_stop":{"unit":"%","size":40,"sizes":[]},"background_gradient_type":"radial","background_gradient_position":"top left","border_border":"double","border_width":{"unit":"px","top":"10","right":"10","bottom":"10","left":"10","isLinked":true},"border_color":"#010102","typography_typography":"custom","typography_font_family":"Roboto","typography_font_weight":"500","text_padding":{"unit":"px","top":"13","right":"13","bottom":"13","left":"13","isLinked":true}},"elements":[],"isInner":false,"widgetType":"button","elType":"widget"}],"isInner":false,"elType":"container"}],"page_settings":[],"version":"0.4","title":"button","type":"page"}
```
Comment (hein-obox) [inline tests/elements-regression/tests/templates/button.json:1]: @rami-elementor My request would be to create a separate PR in which you update the test, and to merge that PR first.

In this way we can see better that we are not breaking something.

I am hesitant to make these changes at the same time.

Wdyt?

Comment (rami-elementor) [reply tests/elements-regression/tests/templates/button.json]: @hein-obox Regression test moved to a separate PR: https://github.com/elementor/elementor/pull/28812

File: core/experiments/manager.php [hunk 2]
```diff
@@ -440,8 +440,7 @@ private function add_default_features() {
 			'title' => esc_html__( 'Optimized Markup', 'elementor' ),
 			'tag' => esc_html__( 'Performance', 'elementor' ),
 			'description' => esc_html__( 'Reduce the DOM size by eliminating HTML tags in various elements and widgets. This experiment includes markup changes so it might require updating custom CSS/JS code and cause compatibility issues with third party plugins.', 'elementor' ),
-			static::TYPE_HIDDEN => true,
-			'release_status' => self::RELEASE_STATUS_DEV,
+			'release_status' => self::RELEASE_STATUS_BETA,
```
Comment (hein-obox) [inline core/experiments/manager.php:443]: The plugin tests are currently failing. I am not sure if that is related to your PR.

**Review by hein-obox — APPROVED** [review]

**Review by arielk — APPROVED** [review]
